hugegraph-core/src/main/java/com/baidu/hugegraph/backend/tx/GraphIndexTransaction.java

```diff
@@ -79,6 +86,18 @@ public GraphIndexTransaction(HugeGraph graph, BackendStore store) {
         assert this.textAnalyzer != null;
     }
 
+    protected Id asyncRemoveIndexLeft(ConditionQuery query,
+                                      HugeElement element) {
+        RemoveLeftIndexCallable callable = new RemoveLeftIndexCallable(query,
+                                                                       element);
+        EphemeralJobBuilder<Object> builder =
+                                    EphemeralJobBuilder.of(this.graph())
+                                                       .name(element.name())
+                                                       .job(callable);
+        HugeTask<?> task = builder.schedule();
+        return task.id();
+    }
+
```

Comment:
prefer to move methods associated with removeIndexLeft to class RemoveLeftIndexCallable